composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "composer/installers",
+    "name": "cross-solution/installers",
```

Comment:
Am 15.04.16 um 22:22 schrieb Nik Samokhvalov:
>
> In composer.json
> <https://github.com/composer/installers/pull/296#discussion_r59935448>:
>
> > @@ -1,5 +1,5 @@
> >  {
> > -    "name": "composer/installers",
> > +    "name": "cross-solution/installers",
>
> What are you doing? :)
>
oh God. I've changed the name of the package to test against my fork.
Did not know that this affects my pull request. I've fixed that in my
fork now.

Thank you for the review.